tests/test_percy_snapshot.py

```diff
@@ -0,0 +1,48 @@
+import base64
+import dash
+import dash_html_components as html
+import os
+from .IntegrationTests import IntegrationTests
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class Tests(IntegrationTests):
+    def test_usage(self):
```

Comment:
Could you add some comments in the testing file about your solution? Just something like "Percy doesn't render Cytoscape charts directly (since they are rendered with canvas & webgl and therefore not in the DOM), so take selenium screenshots, render those screenshots with a simple Dash app, and use percy to compare those screenshots."

Reply:
Will do.